### ISSUE TITLE
Sugary Spire Demo 2 Web Port

### ISSUE THREAD

**Issue by BFDIFirey** [burnedpopcorn/Pizza-Tower-1.1.0-Web-Port#5]
Please Make It (link so you can make it) https://thepizzano.itch.io/sugary-spire-p-rank-build

**Comment by burnedpopcorn** [comment]
I might not be able to spend too much time on it, but I could at least release it. Also, if you don't reply within a week of this posting, I'll consider you not interested and I won't do it (cuz again, time)

**Comment by BFDIFirey** [comment]
but I am interested
![Screenshot 2024-05-13 5 40 24 PM](https://github.com/burnedpopcorn/Pizza-Tower-1.1.0-Web-Port/assets/164089374/dd46988a-d291-4de0-8076-c460366ffe49)

**Comment by burnedpopcorn** [comment]
Alright, but again, you will have to host it yourself, and I might take some time to do it

**Comment by BFDIFirey** [comment]
but how do I make it and use the source code

**Comment by burnedpopcorn** [comment]
You don't need to use the source code, because I'll be the one to compile it, unless you want to do it yourself, which then I ask the question of why you contacted me at all

**Comment by BFDIFirey** [comment]
I don't know how to compile

**Comment by burnedpopcorn** [comment]
So do you want me to do it like we agreed, or do you want to do it yourself. Again, I am open to doing it for you, but you gotta decide

**Comment by BFDIFirey** [comment]
well I'm on Chromebook so you can do it

**Comment by burnedpopcorn** [comment]
Alright

**Comment by BFDIFirey** [comment]
I found a decomp of demo 2 https://github.com/realdealmess/OpenSpire

**Comment by burnedpopcorn** [comment]
Thanks, I'll get on to compiling it

**Comment by BFDIFirey** [comment]
ok!

**Comment by BFDIFirey** [comment]
can I test it out or is it already done?

**Comment by burnedpopcorn** [comment]
I imported it to GameMaker v2023.6.0.92, but after it reported about 300 errors in the project, and when I went to compile it, it did compile, but there was an unavoidable crash, and I have no idea where to begin on fixing this. It is either the project src is really fucked up, or I picked the wrong runtime version, as that can break things

TL;DR, Didn't Work, either a compiler version problem or a source code problem

**Comment by burnedpopcorn** [comment]
Also i used 2023.6.0.92 instead of the recommended 2023.6.0.139 runtime because I could not find it (also thought it wouldn't matter). I'll try to find it somewhere tho

**Comment by BFDIFirey** [comment]
ok

**Comment by BFDIFirey** [comment]
@burnedpopcorn why don't you just upload the game to the gx Games website cause cant you just upload a exe file and make it be a gx game?

**Comment by burnedpopcorn** [comment]
Are you talking about the Noise update port, because its too big for gx.games, and either way I wanted to play it unblocked at my school, and gx.games was blocked in mine

**Comment by BFDIFirey** [comment]
oh well I was gonna say you could've done it with the sugary Spire Demo 2 thing and upload the file but I just figured that gx.games was blocked at your school

**Comment by burnedpopcorn** [comment]
@BFDIFirey, I finally have the Sugary Spire Playtest 3.2 ported to the web browser (if you still care)
still gotta work on it a bit
